supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+== Access token
+A token that can be provided as part of an HTTP request that grants access to the service being invoked on. This is part of the OpenID Connect and OAuth 2.0 specification.
```

Comment:
@ndeevy I don't understand the "on" at the end of the sentence. Would it be correct to say the following:
```suggestion
A token that can be provided as part of an HTTP request that grants access to the service on which the request is  being invoked. This is part of the OpenID Connect and OAuth 2.0 specification.
```